keyboards/torn/torn.c

```diff
@@ -49,6 +49,16 @@ void torn_set_led(uint8_t led, bool state) {
     mcp23018_writeReg(IODIRB, &iodir, 1);
 }
 
+void sleep_led_init(void) {
+}
+
+void sleep_led_enable(void) {
+    oled_off();
+}
+
+void sleep_led_disable(void) {
+}
+
```

Comment:
I would actually recommend: 
```suggestion
void suspend_power_down_kb(void) {
    oled_off();
    suspend_power_down_user();
}
```

This doesn't need a define to enable or anything.